docs/Contributing.asciidoc

```diff
@@ -293,6 +293,17 @@ assets can need a big amount of disk space.
 
 WARNING: Be sure to *clear* that variable when running unit tests locally.
 
+For convenience you can use the `openqa-run` wrapper in to call openQA
+services with `OPENQA_BASEDIR` set to your local development environment with
+the appropriate service account. For example to start the openQA GRU service
```

Comment:
I disagree. The service account this script chooses is not appropriate for local development. I recommend *not* to mix the production/packaged setup with the development environment.

This description also makes it not clear at all to what `OPENQA_BASEDIR` is actually set (because "your local development environment" is not clear in this context, *apparently* it means test files from the current working dir but also *only* if `OPENQA_BASEDIR` is not set anyway).